src/screens/auth/Login.css

```
@@ -0,0 +1,42 @@
+  padding: 0.75rem;
+  margin: 0.75rem 0;
+  border-radius: 0.375rem;
+  border: 0.0625rem solid #ddd;
+  font-size: 1rem;
+}
+
+.login-error {
+  color: red;
+  margin-top: 0.75rem;
+  font-size: 0.875rem;
+  margin: 0px auto;
```

Biome (2.1.2)

[error] Unexpected shorthand property margin after margin-top



(lint/suspicious/noShorthandPropertyOverrides)